assets/airports/eidw.json

```diff
@@ -875,136 +1072,578 @@
-                "28": [["OE", "A30+"], "BAL"],
-                "16": [["DWS77", "A30+"], "DWS81"],
-                "34": [["DWN26", "A30+"], "BAL"]
+                "34": ["OE", ["DWW02", "A40+"]]
+            },
+            "body": [],
+            "exitPoints": {
+                "PELIG": ["PELIG"]
+            },
+            "draw": [
+                ["PELIG"]
+            ]
+        },
+        "PESIT": {
+            "icao": "PESIT",
+            "name": "Pesit",
+            "suffix": {"10":"2E", "16":"1M", "28":"5A", "34":"5G"},
+            "rwy": {
+                "10": [["^DW554", "A30+"], "IRDEX", "ERVAD"],
+                "16": [["^DWS77", "A30+"], "DWS81", "NEPOD"],
+                "28": [["^OE", "A30+"], "BAL", "NEPOD"],
+                "34": [["^DWN26", "A30+"], "BAL", "NEPOD"]
+            },
+            "body": [],
+            "exitPoints": {
+                "PESIT": ["PESIT"]
             },
-            "body": ["NEPOD", "BEPAN"],
-            "draw": [["DW554", "IRDEX", "ERVAD", "NEPOD"], ["OE", "BAL"], ["DWS77", "DWS81", "NEPOD"], ["DWN26", "BAL", "NEPOD", "BEPAN"]]
+            "draw": [
+                ["ERVAD", "PESIT"],
+                ["DWS80", "DW682"],
+                ["OE", "DWW02", "BAL"],
+                ["NEPOD", "PESIT"]
+            ]
         },
         "ROTEV": {
             "icao": "ROTEV",
             "name": "Rotev",
-            "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G"},
+            "suffix": {"10":"6E", "16":"5M", "28":"1A", "34":"1G"},
             "rwy": {
-                "10": [["DW551", "A30+"], "LEMTA"],
-                "28": [["OE", "A30+"], "DWN29"],
-                "16": [["DWS77", "A30+"], "BATED", "ERUDA"],
-                "34": [["DWN26", "A30+"], "DWN29"]
+                "10": [["^DW551", "A30+"], "LEMTA"],
+                "16": [["^DWS77", "A30+"], "BATED", "ERUDA"],
+                "28": [["^OE", "A30+"], "DWN29"],
+                "34": [["^DWN26", "A30+"], "DWN29"]
+            },
+            "body": [],
+            "exitPoints": {
+                "ROTEV": ["ROTEV"]
             },
-            "body": ["ROTEV"],
-            "draw": [["LEMTA", "ROTEV"], ["OE", "DWN29"], ["ERUDA", "ROTEV"], ["DWN29", "ROTEV"]]
+            "draw": [
+                ["LEMTA", "ROTEV"],
+                ["_LIFFY21", "DWN29"],
+                ["ERUDA", "ROTEV"],
+                ["DWN29", "ROTEV"]
+            ]
         },
-        "NEVRI": {
-            "icao": "NEVRI",
-            "name": "Nevri",
-            "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G"},
+        "ROTEV(AB)": {
+            "icao": "ROTEV(AB)",
+            "name": "Rotev",
+            "suffix": {"10":"6F", "16":"5P", "28":"1B", "34":"1H"},
+            "rwy": {
+                "10": ["DW931", "LEMTA"],
+                "16": ["OP", "BATED", "ERUDA"],
+                "28": ["DUB", "DWN29"],
+                "34": ["DUB", "DWN29"]
```

Comment:
Are you sure? ERUDA Doesn't show up on the charts for NEVRI1H:
![eruda](https://user-images.githubusercontent.com/13125311/26904228-dce6e1d4-4bd8-11e7-904b-3f7915977d0b.PNG)